test/ng/directive/inputSpec.js

```diff
@@ -3046,6 +3046,26 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    they('should update the $prop "value" property and attribute after change the "value" property', {
+      input: '<input type="text" ng-value="value">',
+      textarea: '<textarea ng-value="value">></textarea>'
+    }, function(tmpl) {
+      var element = helper.compileInput(tmpl);
+
+      $rootScope.$apply(function() {
+        $rootScope.value = 'something';
+      });
+      expect(element[0].value).toBe('something');
+      expect(element[0].getAttribute('value')).toBe('something');
+
+      helper.changeInputValueTo('newValue');
```

Comment:
@m-amr, basically this is redundant in this test, because we have already tested that the `value` property _does_ have a value. If you wanted to test the scenario your describe, you should edit the input, before setting the value (maybe in a separate test).

<sub>FWIW, I don't think an extra `expect` is needed.</sub>